Resources/Prototypes/Entities/Effects/chemistry_effects.yml

```
@@ -76,6 +76,9 @@
     animationState: foam-dissolve
   - type: Slippery
   - type: StepTrigger
+    triggerGroups:
+      types:
+      - SlipTile
```

yamllint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)